apps/volontulo/tests/common.py

```diff
@@ -73,22 +71,6 @@ def initialize_filled_volunteer_and_organization():
         'volunteer2'
     )
     volunteer_user2.save()
-    user_profile2 = UserProfile.objects.create(user=volunteer_user2)
```

Comment:
@filipgorczynski that's the reason, why travis is failing. Are You able to readd this particular line (without assigning to `user_profile2`)?